js/style/style_layer.js

```diff
@@ -169,6 +169,20 @@ StyleLayer.prototype = util.inherit(Evented, {
         }
     },
 
+    getPaintValueStopZoomLevels: function(name) {
+        var transition = this._paintTransitions[name];
+        if (transition) {
+            return transition.declaration.getStopZoomLevels();
+        } else {
+            return [];
+        }
+    },
+
+    getPaintInterpolationValue: function(name, zoom) {
```

Comment:
Two possible names:
`calculateInterpolationT`
`calculateInterpolationAmount`

I'll also move this out of mapbox-gl-function.